lms/static/js/student_account/views/RegisterView.js

```diff
@@ -18,13 +19,15 @@
 
                 formType: 'register',
 
+                defaultFormErrorsTitle: gettext("We couldn't create your account."),
```

Comment:
Nitpick: single quotes should be used instead of double wherever possible in keeping with the coding style of the repo.